react/queries/categoriesQuery.gql

```diff
@@ -11,6 +11,12 @@ query {
       href
       slug
```

Comment:
You are passing the param `treeLevel: 2` instead of 3.

Reply:
The treeLevel on the API it's a fake (broken?), they ignore the number and always return everything.

Reply:
Changing my answer, the treeLevel works, but don't count the department as one, so if you send 2, will have 2 childrens